zhazam/Modules/Loading/LoadingViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  LoadingViewController.swift
+//  zhazam
+//
+//  Created by Nurbek Ismagulov on 8/20/19.
+//  Copyright © 2019 Nurbek Ismagulov. All rights reserved.
+//
+
+import UIKit
+
+final class LoadingViewController: UIViewController {
+    enum LocalConstants {
```

Comment:
What do you think about removing `Local` prefix and making the whole enum `private`?
I think this is the perfect use case for access modifiers and that's what `private` was built for: to mark certain chunks of code as `local`.